src/more-infos/more-info-climate.html

```diff
@@ -208,6 +238,20 @@
     }.bind(this), 500);
   },
 
+  computeTargetTempHidden: function (stateObj) {
+    return stateObj.attributes.operation_mode.includes('Off');
+  },
+
+  computeHideTempRangeSlider: function (stateObj) {
+    return !stateObj.attributes.operation_mode.includes('Auto');
```

Comment:
Should have listened to me @justweb1 : https://github.com/home-assistant/home-assistant/issues/3378